README.md

```diff
@@ -1,148 +1,43 @@
-# purescript-uri
+# uri
```

Comment:
I'm in favor of capitalizing packages in titles where appropriate (in this case, URI). In the tool that'd be `--display-title URI`.

Reply:
Ok. I'm going to run the following code to update this:
```
contrib-updater generate --title 'URI' --maintainer garyb --display-name '`purescript-uri`'
```
It wasn't clear to me what the difference between `title` and `display-name` was, nor our preference for how to use each.

Reply:
That should be better documented. The short version: 

* `display-title` is used only in headers, when you might display the package in title case. Defaults to the package name, no backticks.
* `display-name` is used when referring to the package informally, like "Thank you for contributing to <display-name>". Defaults to the package name, with backticks (so specifying ```--display-name '`uri`'``` is not necessary)

The package name itself is used in any installation / code instructions (but this is done automatically for you, with no option). Typically I'm only supplying the `--display-title` flag to capitalize the package and rely on the package name for the display name.